### ISSUE TITLE
dark theme is not working 

### ISSUE THREAD

**Issue by shubhamtiwari23** [johnpapa/vscode-winteriscoming#9]
> Please provide us with the following information:
> ---------------------------------------------------------------

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution).macOS(Yosemite ? El Capitan? Sierra ?)

### Versions
  >

### Repro steps
  >

### The log given by the failure.
>

### Mention any other details that might be useful.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by edwinramirez** [comment]
Leaving my feedback as well. Activating light theme works. Activating dark does nothing and leaves you with your currently selected theme.

OS: Mac OS 10.13.3. 
VS Code version: Version 1.20.1

Repro steps: just select the theme from the settings list.

**Comment by wafimardini** [comment]
Hi
Dark theme used to work for me in the begin.
recently stop working on windows & Ubuntu and leaves you with current selected theme.

**Comment by jtsom** [comment]
Pretty much the cause of https://github.com/johnpapa/vscode-winteriscoming/issues/8. There theme .json file has as extraneous comma.

If you want to work around it open the file:

~/.vscode-insiders/extensions/johnpapa.winteriscoming-0.5.0/themes/WinterIsComing-dark-color-theme.json

(If you're not running the Insiders version, substitute .vscode in the above path)

and remove the comma at the end of line 229 (or so):

```
      "scope": [
        "punctuation.decorator",
      ],
```

Should be:
```
      "scope": [
        "punctuation.decorator"
      ],
```

**Comment by kriscoleman** [comment]
Looks like bug was introduced with changes from 03-03-2018. Sadly, theme is now broken. Simple problem too.. sad it's taking more than a couple days to fix a trailing comma...

**Comment by kriscoleman** [comment]
I created a pull request to fix this issue.

**Comment by wafimardini** [comment]
Thanks jtsom Problem solved with extra nasty comma!

**Comment by shubhamtiwari23** [comment]
thanks jtsom problem solved

**Comment by asadferozali** [comment]
Thanks jtsom! Problem Solved.

**Comment by johnpapa** [comment]
Thanks for reporting and to @kriscoleman  for the PR. Now merged and published as 0.5.1

I was travelling back from Finland the past few days and just saw this.

Thanks for using my theme :)
